presto-main/src/main/java/com/facebook/presto/metadata/PolymorphicScalarFunctionBuilder.java

```diff
@@ -249,6 +222,50 @@ public MethodsGroup build()
         }
     }
 
+    public static class ChoiceBuilder
+    {
+        private final Class<?> clazz;
+        private Signature signature;
+        private boolean nullableResult;
+        private List<ArgumentProperty> argumentProperties = emptyList();
```

Comment:
I understand this is pre-existing, but `null` seems like a more natural default value here. In fact, `emptyList` is a terrible choice here due to its ambiguity, although it is unlikely this has any visible impact.